docs/general/ddl/replication.rst

```diff
@@ -1,73 +1,85 @@
-.. _replication:
+.. _ddl-replication:
 
 ===========
 Replication
 ===========
 
-Replication of a table in CrateDB means that each primary shard of a table is
-stored additionally on so called secondary shards. This can be useful for
-better read performance and high availability.
+You can configure CrateDB to *replicate* tables. When a table is replicated,
+table data is stored on multiple primary shards, with each primary shard having
+one or more replica shards.
 
-If not specified, CrateDB creates zero to one replica depending on the number
-of available nodes at the cluster. At a single-node cluster, replicas are set
-to zero to allow fast write operations with the default setting of
-:ref:`sql-create-table-write-wait-for-active-shards`.
+When a primary shard is lost (e.g., due to node failure), CrateDB will promote
+a replica shard to primary. Hence, more table replicas mean a smaller chance of
+permanent data loss (through increased `data redundancy`) in exchange for more
+disk space utilization and intra-cluster network traffic.
 
-.. rubric:: Table of contents
+Replica shards can also improve read performance because more copies of the
+data spread across more nodes. Having more copies of the data on more nodes can
+also increases the opportunities for CrateDB to `parallelize`_ query
+execution.
```

Comment:
based on this and other comments you made, I reworded and expanded the intro:

> You can configure CrateDB to replicate tables. When you configure replication, CrateDB will try to ensure that every table shard has one or more available copies at all times.
>
> When there are multiple copies of the same shard, CrateDB will mark one as the primary shard and treat the rest as replica shards. Write operations always go to the primary shard, whereas read operations can go to any shard. CrateDB continually synchronizes data from the primary shard to all replica shards (through a process known as shard recovery).
>
> When a primary shard is lost (e.g., due to node failure), CrateDB will promote a replica shard to primary. Hence, more table replicas mean a smaller chance of permanent data loss (through increased data redundancy) in exchange for more disk space utilization and intra-cluster network traffic.
> 
> CrateDB allocates every shard to a specific node.CrateDB will try to allocate the primary shard and replica shards to different nodes to maximize durability and resiliency.
>
> Replication can also improve read performance because any increase in the number of shards distributed across a cluster increases the opportunities for CrateDB to parallelize query execution across multiple nodes.